src/Components/Explore/Explore.module.scss

```diff
@@ -0,0 +1,219 @@
+.explorecontainer{
+    background-color:#F3F7FF;
+    height:550px;
+    box-sizing: border-box;
+    .upperhalf{
+        display:flex;
+        justify-content: center;
+        align-items: center;
+        height:50%;
+        .exploretext{
+            font-size: 60px;
+            font-weight:400;
+            font-family: 'Poppins';
+            font-style: normal;
+            color: #404040;
+            line-height: 96px;
+        }
+        .collegetext{
+            font-size: 60px;
+            font-weight:400;
+            font-family: 'Poppins';
```

Comment:
No need to add font-family for all the elements, as its mentioned in index.scss for entire HTML elements